packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/translations/Messages_fr_FR.json

```
@@ -1,6 +1,19 @@
 {
-  "ip_agora_table_header_offer": "Offre",
+  "ip_agora_table_header_offer": "Choisissez votre offre",
   "ip_agora_table_header_quantity": "Quantité",
   "ip_agora_table_header_country": "Localisée(s) en",
-  "ip_agora_table_header_organisation": "Organisation"
+  "ip_agora_table_header_organisation": "Associer une organisation",
+  "ip_agora_failover_ip_title": "Additional IP",
+  "ip_agora_block_ip_title": "Bloc Additional IP",
+  "ip_agora_select_quantity_title": "Choisissez votre quantité",
+  "ip_agora_ip_localisation_title": "IP localisation",
+  "ipv4_select_placeholder": "Selectionner une taille",
+  "license_dashboard_title": "Commander une IP",
+  "ipv4Failover_price_label": "1,50 HT/mois",
```

We missed currency :). better if we could have price from a catalog.
```suggestion
  "ipv4Failover_price_label": "1,50€ HT/mois",
```

---

Okay Mohammed